core/src/test/java/com/linecorp/armeria/server/logging/AnnotatedServiceLogNameTest.java

```diff
@@ -160,7 +162,7 @@ void serviceName_withAnnotation() {
         client.get("/annotation/service-name").aggregate().join();
         assertThat(sctx.config().defaultServiceNaming().serviceName(sctx))
                 .isEqualTo("MyService");
-        assertThat(sctx.config().defaultServiceName()).isEqualTo("MyService");
+        assertThat(sctx.config().defaultServiceName()).isNull();
```

Comment:
I'm not sure... but `@ServiceName()` should override the global naming?

Reply:
Yeah, it should.

Reply:
Oops, I missed that. Let me fix it. 😉